packages/trace-viewer/src/ui/workbenchLoader.css

```diff
@@ -100,6 +100,13 @@ body.dark-mode .drop-target {
 .workbench-loader .logo {
   font-size: 20px;
   margin-left: 16px;
+  display: flex;
```

Comment:
I don't think you need font-size anymore. Also, do you need to align the image in flex?